From: Jonathan Yu <jawnsy@cpan.org>
From: Ansgar Burchardt <ansgar@43-1.org>
Origin: vendor
Forwarded: no
Subject: Change "Æ" to "A" in POD

"Æ" in converted into the macro "\*(AE" which groff does not understand.
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt/Dashes.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt/Dashes.pm
@@ -109,7 +109,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt.pm
@@ -288,7 +288,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt/Basic.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt/Basic.pm
@@ -317,7 +317,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt/GLD.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt/GLD.pm
@@ -146,7 +146,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt/OptionTypeMap.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt/OptionTypeMap.pm
@@ -138,7 +138,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt/Strict.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt/Strict.pm
@@ -77,7 +77,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt/Meta/Attribute.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt/Meta/Attribute.pm
@@ -136,7 +136,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt/Meta/Attribute/NoGetopt.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt/Meta/Attribute/NoGetopt.pm
@@ -98,7 +98,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt/Meta/Attribute/Trait.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt/Meta/Attribute/Trait.pm
@@ -144,7 +144,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
--- libmoosex-getopt-perl.orig/lib/MooseX/Getopt/Meta/Attribute/Trait/NoGetopt.pm
+++ libmoosex-getopt-perl/lib/MooseX/Getopt/Meta/Attribute/Trait/NoGetopt.pm
@@ -94,7 +94,7 @@
 
 =item *
 
-Ævar Arnfjörð Bjarmason <avar@cpan.org>
+Avar Arnfjörð Bjarmason <avar@cpan.org>
 
 =item *
 
